ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py

```diff
@@ -0,0 +1,66 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy.functional.backends.torch as ivy_torch
+
+
+# flip
+@given(
+    dtype_value=
+        helpers.dtype_and_values(
+            available_dtypes=tuple(
+                set(ivy_np.valid_float_dtypes).intersection(
+                    set(ivy_torch.valid_float_dtypes))),
+            shape=st.shared(
+                helpers.get_shape(
+                    min_num_dims=1
+                ),
+                key='shape'
+            ),
+        ),
+    dims=
+        helpers.get_axis(
+            shape=st.shared(
+                helpers.get_shape(
+                    min_num_dims=1
+                ),
+                key='shape'
+            ),
+            min_size=1,
+            max_size=1,
+        ),
+    as_variable=st.booleans(),
+    with_out=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="functional.frontends.torch.flip"),
+    native_array=st.booleans(),
+    data=st.data(),

```

Comment:
We don't need to use `data` here currently, please remove it from here and from the function signature below.